cmd/juju/controller/destroy.go

```diff
@@ -119,6 +121,16 @@ WARNING! This command will destroy the %q controller.
 This includes all machines, applications, data and other resources.
 `[1:]
 
+var destroySysMsgWithDetails = `
+WARNING! This command will destroy the %q controller.
+This includes all data and other resources:
+ - %d model(s) will be destroyed
+  - model list: %q
+ - %d machine(s) will be destroyed
+ - %d application(s) will be removed
+ - %d storage(s) will be %s
```

Comment:
I think this should be split between volumes and filesystems

Reply:
agreed